Aztec/Classes/Libxml2/DocumentObjectModel.swift

```diff
@@ -226,12 +226,12 @@ extension Libxml2 {
         ///     - value: the value found in a `NSLinkAttributeName` attribute.
         ///     - range: the range the style will be applied to.
         ///
-        private func applyStyle(linkValue value: AnyObject, to range: NSRange) {
-            if let urlValue = value as? NSURL {
+        fileprivate func applyStyle(linkValue value: Any, to range: NSRange) {
```

Comment:
The interface for attributes on attributed string changed from [String:AnyObject] to [String:Any]